packages/web-components/src/components/video-player/video-player-composite.ts

```diff
@@ -191,6 +198,7 @@ class DDSVideoPlayerComposite extends HybridRenderMixin(HostListenerMixin(LitEle
         duration="${ifNonNull(duration)}"
         ?hide-caption=${hideCaption}
         name="${ifNonNull(caption || name)}"
+        video-description="${ifNonNull(customVideoDescription)}"
```

Comment:
I see `video-description` is a new attribute being passed into `video-player`,  but looking at the `video-player` component, where is the attribute being used?